tests/controllers/sib-deletion/chainsaw-test.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: sib-deletion
+spec:
+  timeouts:
+    assert: 10s
+  steps:
+
+  - name: step-01 
+    try:
+      - apply:
+          # file is relative to the test folder
+          file: ../../../examples/namespaced/dns-manipulation-si-sib.yaml
+      - assert:
+          file: ../../../examples/namespaced/dns-manipulation-si-sib.yaml
+  - name: step-02
+    try:
+      - delete:
+          ref:
+            apiVersion: intent.security.nimbus.com/v1
+            kind: SecurityIntentBinding
+            namespace: $(namespace)
+            name: dns-manipulation-binding
+          expect:
+          - match:
+              apiVersion: intent.security.nimbus.com/v1
+              kind: SecurityIntentBinding
+              namespace: $(namespace)
+              name: dns-manipulation-binding
+            check:
+              ($error != null): true
```

Comment:
Here you're only checking SIB deletion, we also need to check the deletion for corresponding NP.